src/main/java/uk/gov/hmcts/divorce/common/event/SubmitClarification.java

```diff
@@ -46,8 +55,9 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             .description("Submit clarification for conditional order")
             .showSummary()
             .showEventNotes()
+            .aboutToStartCallback(this::aboutToStart)
             .aboutToSubmitCallback(this::aboutToSubmit)
-            .grant(CREATE_READ_UPDATE, APPLICANT_1_SOLICITOR, CREATOR, APPLICANT_2)
+            .grant(CREATE_READ_UPDATE_DELETE, APPLICANT_1_SOLICITOR, CREATOR, APPLICANT_2)
```

Comment:
Note to reviewer: the delete permission is required to clear the list of uploaded documents in the about to start callback